### ISSUE TITLE
Tag exosuit instances

### ISSUE THREAD

**Issue by peterdenham** [jasonswett/exosuit#8]
What would you think about tagging instances created by exosuit? and only allow instances/resources that are tagged as such to be managed by the cli tool? i.e. filter out any other instances that could be in a users' aws account except exosuit specific ones.

**Comment by jasonswett** [comment]
This is a really good idea. Do you have an idea for how it would work? Maybe any Exosuit-launched instance automatically gets tagged and other instances can get tagged as Exosuit-manageable instances manually?

**Comment by peterdenham** [comment]
Yep, tagging instances on launch sounds ideal. I would refer to their tagging strategies guide for picking something suitable - https://aws.amazon.com/answers/account-management/aws-tagging-strategies/

Once instances are tagged, you could filter on them and/or set IAM policy for access control to exosuit resources.

As an extension, we could think about further context / namespace tags per project when multiple projects are set up within an aws account.

**Comment by jasonswett** [comment]
I would vote that each instance gets (at least) two tags:
- One to designate that it's an Exosuit-managed instance
- One to associate the instance with a certain app

BTW I'm imagining that Exosuit will have the concept of an "app", analogous to the Heroku concept of apps. Each app will have a name with a randomly-generated phrase (for example, the app I'm currently working on is called "intolerable-awful-river").

I'm thinking for simplicity, the Exosuit CLI will only work with one app at a time, and that the current app will be stored under an `app_name` entry in `.exosuit/config.yml`.
